src/dataDisplay/chip/chip.jsx

```diff
@@ -0,0 +1,228 @@
+                        backgroundColor: palette.purple[600],
+                    },
+                },
+            },
+            '&$colorRed': {
+                backgroundColor: palette.red[500],
+                '& $deleteIcon': {
+                    backgroundColor: palette.red[500],
+                    boxShadow: `inset 1px 0 0 ${palette.red[600]}`,
+                    '&:hover': {
+                        backgroundColor: palette.red[600],
+                    },
+                },
+            },
+            '&$colorRedOrange': {
+                backgroundColor: palette.redOrange[500],
+                '& $deleteIcon': {
+                    backgroundColor: palette.redOrange[500],
+                    boxShadow: `inset 1px 0 0 ${palette.redOrange[600]}`,
+                    '&:hover': {
+                        backgroundColor: palette.redOrange[600],
+                    },
+                },
+            },
+            '&$colorSky': {
+                backgroundColor: palette.sky[500],
+                '& $deleteIcon': {
+                    backgroundColor: palette.sky[500],
+                    boxShadow: `inset 1px 0 0 ${palette.sky[600]}`,
+                    '&:hover': {
+                        backgroundColor: palette.sky[600],
+                    },
+                },
+            },
+            '&$colorTeal': {
+                backgroundColor: palette.teal[500],
+                '& $deleteIcon': {
+                    backgroundColor: palette.teal[500],
+                    boxShadow: `inset 1px 0 0 ${palette.teal[600]}`,
+                    '&:hover': {
+                        backgroundColor: palette.teal[600],
+                    },
+                },
+            },
+            '&$disabled': {
+                opacity: 1,
+                '& $deleteIcon': {
+                    backgroundColor: palette.grey[300],
+                    boxShadow: `inset 1px 0 0 ${palette.grey[400]}`,
+                },
+            }
+        },
+    };
+});
+
+/**
+ * Chips are compact elements that represent an input, attribute, or action.
+ */
+function Chip(props) {
+    const {
+        className,
+        color,
+        deleteIcon,
+        disabled,
+    } = props;
+
+    const classes = useStyles(props);
+
+    const rootClasses = ClassNames(
+        UI_CLASS_NAME,
+        BEM_CHIP,
+        className,
+        {
+            [classes.colorGreen]: !disabled && color === 'green',
+            [classes.colorOrange]: !disabled && color === 'orange',
+            [classes.colorPink]: !disabled && color === 'pink',
+            [classes.colorPurple]: !disabled && color === 'purple',
+            [classes.colorRed]: !disabled && color === 'red',
+            [classes.colorRedOrange]: !disabled && color === 'redOrange',
+            [classes.colorSky]: !disabled && color === 'sky',
+            [classes.colorTeal]: !disabled && color === 'teal',
+            [classes.colorTeal]: !disabled && color === 'teal',
```

Comment:
We don't need this twice

Reply:
dang, how did this happen. You get a teal, you get a teal...

Reply:
Fixed.

Reply:
Resolving.